app.js

```diff
@@ -179,15 +175,9 @@ const display = {
 };
 
 const compute = function(currentNumber, previousNumber, operation) {
-    if(operation === '+') {
-        logic.add(previousNumber, currentNumber);
-    } else if (operation === '-') {
-        logic.subtract(previousNumber, currentNumber);
-    } else if (operation === 'X') {
-        logic.multiply(previousNumber, currentNumber);
-    } else if (operation === '÷') {
-        logic.divide(previousNumber, currentNumber);
-    }
+  if (Object.keys(logic).includes(operation)) {
+    logic[operation](previousNumber, currentNumber);
```

Comment:
I want you to pay special attention to this line. So, `logic` is an `Object` full of functions, right? And we're picking the function we want by doing `logic[function_name]`. Is there anywhere else in this file that we could follow the same approach?